src/styles/blocks/star.scss

```diff
@@ -0,0 +1,32 @@
+// .stars {
+//   display: flex;
+//   align-self: flex-start;
```

Comment:
Don't forget to delete comments that you don't need anymore

Reply:
done